src/containers/Results/Results.css

```diff
@@ -69,5 +89,9 @@ tr:hover {
 
 .results-bottom button {
   margin-top: 0%;
-  margin-bottom: 5%;
+  margin-bottom: 1%;
 }
+
+.savings-display-bar {
+    margin-top: 3em;
+}
```

Comment:
I like the new results table styling, especially since it is now in alignment with the color palette!